Milchkaffee.java

```diff
@@ -1,7 +1,14 @@
 // "Das haben wir schon immer getrunken
 // und das werden wir auch weiterhin trinken!"
-class Milchkaffee{
+
+class Milchkaffee {
+  private String nameWithPrice(String price) {
+    return "Milchkaffee - " + price;
+  }
+
   public static void main(String[] args){
-    System.out.println("Milchkaffee - 1,00€");
+    Milchkaffee milchkaffee = new Milchkaffee();
```

Comment:
https://www.heise.de/developer/artikel/Typinferenz-fuer-lokale-Variablen-in-Java-10-Teil-1-4095599.html
`var milchkaffee = new Milchkaffee()`

Reply:
I thought about that, but is it idiomatic yet? :D

Reply:
No, make it Java 1.5 compatible